zulip/README.md

```diff
@@ -23,9 +23,9 @@ as we make any changes to them.
 The easiest way to use these API bindings is to base your tools off
 of the example tools under zulip/examples/ in this distribution.
 
-If you place your API key in the config file `~/.zuliprc` the Python
-API bindings will automatically read it in. The format of the config
-file is as follows:
+If you place your API key in the config file `~/.zuliprc` or './.zuliprc'
```

Comment:
It is more explicit to say `$PWD/.zuliprc` than `./.zuliprc`.

Reply:
Done.